pystiche_papers/johnson_alahi_li_2016/_data.py

```diff
@@ -28,28 +26,6 @@
 def content_transform(
     edge_size: int = 256, multiple: int = 16, impl_params: bool = True,
 ) -> transforms.ComposedTransform:
-    class TopLeftCropToMultiple(transforms.Transform):
```

Comment:
This was too ambitious. I don't think another paper will use this transform again. If time proves me wrong, we handle it then.